lib/runners/workspace/git.rb

```diff
@@ -1,5 +1,17 @@
 module Runners
   class Workspace::Git < Workspace
+    def range_git_blame_info(path_string, start_line, end_line)
+      base = git_source.base
+      head = git_source.head
+      if base && head
+        shell = Shell.new(current_dir: git_directory, trace_writer: trace_writer, env_hash: {})
+        stdout, _ = shell.capture3!("git", "blame", "-p", "-L", "#{start_line},#{end_line}", "#{base}...#{head}", "--", path_string)
```

Comment:
How about suppressing verbose output to trace as follow?

```suggestion
        stdout, _ = shell.capture3!("git", "blame", "-p", "-L", "#{start_line},#{end_line}", "#{base}...#{head}", "--", path_string, trace_stdout: false, trace_stderr: false)
```

Reply:
Oh, I forgot the arguments `:trace_stdout` and `:trace_stderr`. Thank you.